MagazineLayout/Public/Types/MagazineLayoutVerticalLayoutDirection.swift

```diff
@@ -0,0 +1,34 @@
+// Created by bryankeller on 12/8/23.
+// Copyright © 2023 Airbnb Inc. All rights reserved.
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/// The vertical layout direction of items in the collection view. This property changes the behavior of scroll-position-preservation when
+/// performing batch updates or when the collection view's bounds changes.
+public enum MagazineLayoutVerticalLayoutDirection {
+
+  /// The vertical layout direction of items goes from top to bottom. Items inserted at the top push content down. When scrolled to the
```

Comment:
mention that this is the standard way UICollectionViewLayouts work?

Reply:
I basically say that here https://github.com/airbnb/MagazineLayout/pull/114/files#diff-a65dd25675c8fc47b5b55e8f3d48d4f20b50d15929e0269692e506ec8aa7cc4bR24